src/firmware/firmware.ino

```diff
@@ -2233,6 +2250,20 @@ bool saveHaAlarmAuto(int newMode) {
   return true;
 }
 
+
+
+bool saveHaNightMode(bool newState) {
+  nightMode = newState;
+  Serial.print("nightMode: ");
+  Serial.println(nightMode ? "true" : "false");
+#if HA_ENABLED && DISPLAY_ENABLED
```

Comment:
Why do we depend on DISPLAY_ENABLED here? We can use HA integration even without display.

Reply:
++